### DISCUSSION TITLE
Login and autocomplete with ID + username + password

### DISCUSSION THREAD

**Discussion by mdibarra** [keepassxreboot/keepassxc#8957] [Q&A] [answered]
Hi all, this bank URL https://onlinebanking.bancogalicia.com.ar/login require three methods to login: Personal ID, Username and Password, currently on 2.7.4 version I just have the possibility of storage the username and password. 
There is any way to cover the three option to do a login with the autocomplete function?

**Comment by droidmonkey** [comment, marked as answer]
Use advanced attributes. Read the user guide for more information.

**Reply by mdibarra** [reply]
Thanks for your answer. I have read the documentation and I have not been able to understand how to make the third attribute be entered automatically.

**Reply by droidmonkey** [reply]
Apologies we haven't yet rolled that into the user guide, it was in the FAQ: https://keepassxc.org/docs/#faq-browser-string-fields

**Reply by mdibarra** [reply]
Thank you Jonathan.
